scripts/gitlab-server/gitlab-runner-setup.sh

```diff
@@ -33,34 +34,38 @@ shArgs.arg "CERT_PATH" -c --cert-path PARAMETER true
 shArgs.arg "GITLAB_TOKEN" -gt --gitlab-token PARAMETER true
 shArgs.arg "GITLAB_DOMAIN" -gd --gitlab-domain PARAMETER true
 shArgs.arg "SERVER_INTERNAL_IP" -si --server-ip PARAMETER true
+shArgs.arg "CONFIG_PATH" -cp --config-path PARAMETER true
 
 shArgs.parse $@
 
 main(){
     print_banner
     verify_tool_exists "az"
     check_inputs
+
     check_az_is_logged_in
 
     if [ "$MODE_FULL" == true ]; then
         _debug "Running in full mode."
-        for i in {1..5}; do
-            local vmName="$VM_PREFIX-$i"
-            create_single_vm "$vmName"
+        for level in {1..5}; do
```

Comment:
Should we switch this out to 0..4 or is that irrelevant?

Reply:
Good call, I will make the change.